### PR TITLE
Dec 21 Add on onboarding strings for premium users

### PR DESCRIPTION
**Pull request by codemist**
New strings for premium onboarding.

Preview:
image

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -481,20 +481,27 @@ onboarding-premium-domain-title = Use a custom domain for sharing aliases
 onboarding-premium-domain-description = With a custom domain, you can make your “newsletter” alias different from your “shopping” alias. 
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
+onboarding-premium-title-detail = With Firefox Relay Premium you can:
```
Comment (peiying2) [inline en/app.ftl:484]: Firefox Relay Premium is a brand

**Review by peiying2 — CHANGES_REQUESTED** [review]
Please use placeable for brand name

**Comment by Vinnl** [comment]
As a heads-up, I added some strings here that were approved via #49, so that we can merge them in one go and only send a single notification to translators.

**Comment by peiying2** [comment]
@Vinnl do you mind making the change here? Looks like she could be off today.

**Comment by Vinnl** [comment]
@peiying2 Sure, just pushed a fix!

**Comment by peiying2** [comment]
> @peiying2 Sure, just pushed a fix!

Thank you for the quick response. Nitpick question, do you think we should add a comma, so it reads `With xyz, you can:`

**Comment by Vinnl** [comment]
Either sounds fine to my (non-native) ear, so I'd defer to content (who I presume suggested the current version).

**Review by peiying2 — APPROVED** [review]
Okie Dokie.

**Comment by codemist** [comment]
Thanks @Vinnl !!

**Pull request merged**